tests/patch_time.py

```diff
@@ -20,10 +16,21 @@ def _monotonic() -> float:
     return time.monotonic()
 
 
-# Replace partial functions which are not found by freezegun
+# Before importing anything else, replace partial dt_util.utcnow
+# with a regular function which can be found by freezegun
+from homeassistant import util  # noqa: E402
+from homeassistant.util import dt as dt_util  # noqa: E402
+
 dt_util.utcnow = _utcnow  # type: ignore[assignment]
```

Comment:
```suggestion

util.utcnow = util.dt.utcnow = _utcnow  # type: ignore[assignment]
```

Reply:
I'd prefer to be explicit here instead of doing multiple assignments on one line